Add 5.17.1 release notes (#2148)
sensu/sensu-docs: content/sensu-go/5.17/release-notes.md

---

## 5.17.1 release notes

**January 31, 2020** &mdash; The latest release of Sensu Go, version 5.17.1, is now available for download.

> This release fixes a web UI issue that cleared selected filters when sorting an event list and a bug that prevented certain `.tar` assets from extracting and. It also includes sensuctl configuration improvements.

Author: ```suggestion
This release fixes a web UI issue that cleared selected filters when sorting an event list and a bug that prevented certain `.tar` assets from extracting.
It also includes sensuctl configuration improvements.
```

See the [upgrade guide][1] to upgrade Sensu to version 5.17.1.

**IMPROVEMENTS:**

- Asset names may now include capital letters.
- Running the `sensuctl configure` command now resets the sensuctl cluster configuration.
- When you use `--trusted-ca-file` to configure sensuctl, it now detects and saves the absolute file path in the cluster configuration.

**FIXES:**

- ([Commercial feature][106]) In the [web UI][107], polling is re-enabled for the silences view.

> - Fixed a bug that prevented `.tar` assets that contain hardlinked files from extracting.

Author: ```suggestion
- Fixed a bug that prevented `.tar` assets from extracting if they contain hardlinked files.
```

---

## 5.17.1 release notes

**January 31, 2020** &mdash; The latest release of Sensu Go, version 5.17.1, is now available for download.
This release fixes a web UI issue that cleared selected filters when sorting an event list and a bug that prevented certain `.tar` assets from extracting.
It also includes sensuctl configuration improvements.

See the [upgrade guide][1] to upgrade Sensu to version 5.17.1.

**IMPROVEMENTS:**

- Asset names may now include capital letters.
- Running the `sensuctl configure` command now resets the sensuctl cluster configuration.
- When you use `--trusted-ca-file` to configure sensuctl, it now detects and saves the absolute file path in the cluster configuration.

**FIXES:**

> - ([Commercial feature][106]) In the [web UI][107], polling is re-enabled for the silences view.

Reviewer: I believe this change should be described differently. As I understand it from https://github.com/sensu/sensu-enterprise-go/issues/853, users may have noticed silencing entries lingering in the web UI after they had been removed. The fix ensures that the web UI is updated when a silencing entry is removed or expires.

> - ([Commercial feature][106]) In the [web UI][107], polling is re-enabled for the silences view.

Author: ```suggestion
- ([Commercial feature][106]) When a silencing entry expires or is removed, it is also removed from the silences view in the [web UI][107].
```
